dataactcore/config_example.yml

```diff
@@ -137,6 +137,7 @@ logging:
             - use_aws
             - local
             - aws_bucket
+            - certified_bucket
```

Comment:
aws_bucket wasn't in there either so I just followed that, do we want to add it?